apps/web/docker/Dockerfile

```diff
@@ -86,4 +81,4 @@ ENV NEXT_TELEMETRY_DISABLED=1
 
 EXPOSE $PORT
 
-CMD node apps/web/server.js -p $PORT -h $HOSTNAME
+CMD node .next/standalone/apps/web/server.js -p $PORT -h $HOSTNAME
```

Comment:
web dockerfile is almost exactly the same than the others, but not exactly because of some next peculiarities